src/eureka/lib/apphot.py

```diff
@@ -475,81 +82,69 @@ def apphot(meta, image, ctr, photap, skyin, skyout, betahw, targpos,
 
     # Check for skyfrac violation
     # FINDME: include NaNs and zero errors
```

Comment:
Do we still need this FINDME?

Reply:
We were already counting NaNs, but not zero errors, so I added that above and have now removed this FINDME